mldsa/poly.c

```diff
@@ -359,18 +228,8 @@ __contract__(
   return ctr;
 }
 
-/*************************************************
- * Name:        poly_uniform
- *
- * Description: Sample polynomial with uniformly random coefficients
- *              in [0,MLDSA_Q-1] by performing rejection sampling on the
- *              output stream of SHAKE128(seed|nonce)
- *
- * Arguments:   - poly *a: pointer to output polynomial
- *              - const uint8_t seed[]: byte array with seed of length
- *MLDSA_SEEDBYTES
- *              - uint16_t nonce: 2-byte nonce
- **************************************************/
+#define POLY_UNIFORM_NBLOCKS \
```

Comment:
This macro appears to be defined twice now - lines 191 and 231. I think the earlier one is needed so it can be used in contracts?

Reply:
Good catch - that was moved in #94 and the rebase didn't work as intended.
I fixed it.